packages/client/src/knock.ts

```diff
@@ -74,11 +103,46 @@ class Knock {
 
   // Used to teardown any connected instances
   teardown() {
+    if (this.tokenExpirationTimer) {
+      clearTimeout(this.tokenExpirationTimer);
+    }
     if (!this.apiClient) return;
     if (this.apiClient.socket) {
       this.apiClient.socket.disconnect();
     }
   }
+
+  private async maybeScheduleUserTokenExpiration(
+    callbackFn: UserTokenExpiringCallback,
+    timeBeforeExpirationInMs: number = 1000,
```

Comment:
I think we should set this to be 30_000 as a default to give plenty of time to refresh